demo/src/app.js

```diff
@@ -394,6 +412,13 @@ const AppContent = ({ language, onLanguageClick }) => {
         []
     );
 
+    const CustomTreeViewFinder =
+        stylesProvider === 'emotion'
```

Comment:
Why do we keep both way ?

Reply:
I made a backwards compatible migration. It's still possible to generate classnames with jss, and then use the classes APIs to apply the classes. This is to make migrations easier. It's what mui did. So in the demo, we can test both modes. If mui removes the classes API, we will have to drop support for js, but for now it's easy to support